frontend/src/scenes/insights/utils.tsx

```diff
@@ -253,7 +253,11 @@ export function formatBreakdownLabel(
             ? formatPropertyValueForDisplay(breakdown, breakdown_value)?.toString() ?? 'None'
             : breakdown_value.toString()
     } else if (typeof breakdown_value == 'string') {
-        return breakdown_value === 'nan' ? 'Other' : breakdown_value === '' ? 'None' : breakdown_value
+        return breakdown_value === 'nan' || breakdown_value === '$$_posthog_breakdown_other_$$'
```

Comment:
Magic strings 🧙 🪄

Reply:
I know. 🙈

Yet the complexity of changing this to a dict of some sort throughout the query, backend and frontend layers was too frightening... We're unlikely to have conflicts with real world data with a string like this, so 🪄 🌈 🦄